regression/pages/studio/login_studio.py

```diff
@@ -0,0 +1,19 @@
+from bok_choy.promise import EmptyPromise
+from edxapp_pages.studio.login import LoginPage
+
+
+class StudioLogin(LoginPage):
+    def login(self, email, password):
+        """
+        Attempt to log in using `email` and `password`.
+        """
+
+        self.q(css='input#email').fill(email)
+        self.q(css='input#password').fill(password)
+        self.q(css='button#submit').first.click()
+
+        # Ensure that we make it to Studio home page
+        EmptyPromise(
```

Comment:
instead of `EmptyPromise` you should use `wait_for_page()` on a home page object.